.env

```diff
@@ -1,7 +1,7 @@
 PORT=3000
 ETHCONNECT_URL=http://127.0.0.1:5102
-ETHCONNECT_INSTANCE=/contracts/<Address of ERC20 Factory>
+ETHCONNECT_INSTANCE=/contracts/<Address of ERC20 Factory>              # if ethconnect has remote registry use /instances/<Address of ERC20 Factory>
 ETHCONNECT_TOPIC=token
 ETHCONNECT_PREFIX=fly
 AUTO_INIT=true
-CONTRACT_ABI_ID=<ABI ID of ERC20 contract instance>
\ No newline at end of file
+ETHCONNECT_CONTRACT_URI=abis/<ABI ID of ERC20 contract instance>      # if ethconnect has remote registry use gateways/<Gateway name of ERC20 contract>
```

Comment:
Should this match the `instance` env var and be `/abis/{id}` instead of `abis/{id}`?

Reply:
I think that would be cleanest